README.md

```diff
@@ -95,28 +97,151 @@ features = extract_features("test_logs/Sepsis.xes")
+'end_activities_q3': 53.5
+'end_activities_skewness': 2.004413358907822
+'end_activities_std': 112.91400014423114
+'end_activities_variance': 12749.57142857143
+'execution_time_time_coefficient_variation': 8.499578843161144
+'execution_time_time_entropy': 6.221052534222753
+'execution_time_time_geometric_mean': 199.88320191111325
+'execution_time_time_geometric_std': 127.92792986844444
+'execution_time_time_harmonic_mean': 0.0
+'execution_time_time_hist0': 2.7448347367316943e-07
+'execution_time_time_hist1': 1.002762607241735e-09
+'execution_time_time_hist2': 6.928178013670171e-10
+'execution_time_time_hist3': 3.2817685327911313e-10
+'execution_time_time_hist4': 3.2817685327911313e-10
+'execution_time_time_hist5': 2.552486636615327e-10
+'execution_time_time_hist6': 9.116023702197587e-11
+'execution_time_time_hist7': 5.469614221318552e-11
+'execution_time_time_hist8': 7.292818961758069e-11
+'execution_time_time_hist9': 7.292818961758069e-11
+'execution_time_time_iqr': 18623.25
+'execution_time_time_kurtosis': 250.48825320470718
+'execution_time_time_kurtosis_hist': 5.110914600502133
+'execution_time_time_max': 36051318.0
+'execution_time_time_mean': 169759.47397134217
+'execution_time_time_median': 188.0
+'execution_time_time_min': 0.0
+'execution_time_time_mode': 0.0
+'execution_time_time_q1': 0.0
+'execution_time_time_q3': 18623.25
+'execution_time_time_skewness': 14.528527518337814
+'execution_time_time_skewness_hist': 2.666603580180752
+'execution_time_time_std': 1442884.0333930943
+'execution_time_time_variance': 2081914333820.724
+'kurtosis_variant_occurrence': 217.44268017168216
+'log': 'SEPSIS'
+'mean_variant_occurrence': 1.2411347517730495
 'n_traces': 1050
+'n_unique_activities': 16
+'n_unique_end_activities': 14
+'n_unique_start_activities': 6
 'n_unique_traces': 846
+'ratio_most_common_variant': 0.03333333333333333
+'ratio_top_10_variants': 0.2742857142857143
+'ratio_top_1_variants': 0.12
+'ratio_top_20_variants': 0.35523809523809524
+'ratio_top_50_variants': 0.5971428571428572
+'ratio_top_5_variants': 0.21523809523809523
+'ratio_top_75_variants': 0.7980952380952381
 'ratio_unique_traces_per_trace': 0.8057142857142857
-'trace_len_min': 3
-'trace_len_max': 185
-'trace_len_mean': 14.48952380952381
-'trace_len_median': 13.0
-'trace_len_mode': 8
-'trace_len_std': 11.470474925273926
-'trace_len_variance': 131.57179501133788
-'trace_len_q1': 9.0
-'trace_len_q3': 16.0
-'trace_len_iqr': 7.0
-'trace_len_geometric_mean': 12.281860759040903
-'trace_len_geometric_std': 1.7464004837799154
-'trace_len_harmonic_mean': 10.47731701485374
-'trace_len_skewness': 7.250526815880918
-'trace_len_kurtosis': 87.0376906898399
+'remaining_time_time_coefficient_variation': 1.8886403422401359
+'remaining_time_time_entropy': 8.55331137332654
+'remaining_time_time_geometric_mean': 224736.22203397762
+'remaining_time_time_geometric_std': 70.1715364379747
+'remaining_time_time_harmonic_mean': 0.0
+'remaining_time_time_hist0': 2.1808895944590443e-07
```

Comment:
Please include the features with condensed time_based knowledge only instead of e.g. x_y_hist1, x_y_hist2, x_y_hist3, ..., x_y_histn. I think those are less interpretable for humans, so that including them would diminish that argument in our paper. 🤔